x-pack/plugins/enterprise_search/server/lib/indices/fetch_indices.ts

```diff
@@ -38,6 +38,7 @@ export const mapIndexStats = (
   return {
     aliases,
     health: indexStats?.health,
+    hidden: indexData.settings?.index?.hidden === 'true',
```

Comment:
gotta fix tests now (and adding a case to check for this condition might be good, too)